src/gui/tray/ActivityList.qml

```diff
@@ -36,6 +45,20 @@ ScrollView {
         currentIndex: -1
         interactive: true
 
+        Timer {
+            id: repeatUpFlickTimer
+            interval: 50
+            running: controlRoot.scrollingToTop
+            repeat: true
+            onTriggered: {
+                if (!activityList.atYBeginning) {
+                    activityList.flick(0, 10000)
+                } else {
+                    controlRoot.scrollingToTop = false
+                }
+            }
+        }
+
```

Comment:
I think a better way to do this would be to do so by manipulating the y position of the `ScrollView` directly with an `Animation` type. You can either make the scrollview wrap a `Flickable` or you can operate on the Scrollbar, either way it will prevent you from having to fire repeating timers to flick the view up

Reply:
@claucambra I tried changing the `contentY` position, however, this does not work as expected, the list can not be scrolled to the very top, it gets scrolled just a bit but is still too fat away from the top, any ideas?

Reply:
Maybe you can try setting this through the attached `ScrollBar` property?